.atomist/tests/handlers/command/ListActiveIssuesSteps.ts

```diff
@@ -0,0 +1,22 @@
+import { Given, When, Then,
+         CommandHandlerScenarioWorld } from "@atomist/rug/test/handler/Core";
+import { Execute, Respondable } from '@atomist/rug/operations/Handlers';
+
+const url = `https://api.github.com/repos/atomist/rug/issues?sort=comments&state=open`;
+
+Given("nothing", f => { });
+
+When("the ListActiveIssues is invoked", (world: CommandHandlerScenarioWorld) => {
+    const handler = world.commandHandler("ListActiveIssues");
+    world.invokeHandler(handler, {
+        repo: "rug",
+        owner: "atomist"
+    });
+});
+
+Then("you get a plan which will retrieve the list of open and active issues", (world: CommandHandlerScenarioWorld) => {
+    const instruction = world.plan().instructions[0] as Respondable<Execute>;
```

Comment:
The tests are not actually shown or discussed in the tutorial text, so perhaps this discussion can be had offline.